src/topp/PeptideIndexer.cpp

```diff
@@ -600,10 +598,10 @@ class TOPPPeptideIndexer :
 
         String acc = proteins[i].identifier;
         // check for duplicate proteins
+        // @TODO: what happens if there are duplicate sequences with different accessions?
         if (acc_to_prot.has(acc))
         {
-          LOG_WARN << "PeptideIndexer: Warning, protein identifiers should be unique to a database. Identifier '" << acc << "' found multiple times.\n";
-          has_DB_duplicates = true;
+          LOG_WARN << "PeptideIndexer: Warning, protein identifiers should be unique to a database. Identifier '" << acc << "' found multiple times." << endl;
```

Comment:
I didn't exactly keep the old code, but I put in a version that doesn't use "endl" repeatedly.